CONTRIBUTING.md

```diff
@@ -9,26 +11,52 @@ In Mac OS X:
     $ brew install node
     $ export PATH=$PATH:/usr/local/share/npm/bin
     $ source .bash_profile
+    
+In Linux:
 
-### To start development
+    $ apt-get install nodejs
+    
+In Windows: [Download the 32-bit version of Nodejs installer.](http://nodejs.org/download/)
+
+#### Installing Ruby
+
+In Mac OS X:
 
-You need to install grunt-cli, bower and karma first
+    $ brew install ruby
+    $ export PATH=$PATH:/usr/local/opt/ruby/bin
+    $ source .bash_profile
+    
+In Linux:
 
-    $ npm install -g grunt-cli bower karma
+    $ apt-get install ruby1.9.3 rubygems
+    
+In Windows: [Follow the installation instructions here to install Ruby and RubyGems](https://github.com/bethesque/pact-mock_service/wiki/Installing-the-pact-mock_service-gem-on-Windows)
 
-Inside the project folder, simply run `npm install` and `bundle install` to install the other dependencies.
+#### Installing Bundle & pact-mock-service
 
-### During Development
+Run the following commands (works in all OS'):
 
-To run gulp tasks:
+    $ gem i bundler
 
-    $ gulp
+### To start development
 
-or if you don't want to install the gulp-cli globally:
+You need to install global binaries to build and test the code:
 
```

Comment:
I would think that installing gulp, bower and karma as global or local is the user's choice, is it not? There's nothing in the pact dsl that requires them to be global is there?